_docs/_latest/setup/upgrade.md

```diff
@@ -5,72 +5,105 @@ order: 00
 
 # {{ page.title }}
 
-This guide explains how to use the Forseti upgrade tool.
+This guide explains how to upgrade your Forseti instance.
 
-## Before you begin
+---
+
+{% capture 1x_upgrade %}
 
-Before you upgrade Forseti, you'll need the following:
+## Important notes
 
- * A G Suite super admin account to complete the [Domain-wide delegation steps]({% link _docs/latest/configure/gsuite.md %}). 
-Forseti 2.0 requires G Suite being enabled, so you'll need this before you start the installation.
+ * We don't support data migration from v1 to v2, so you will need to archive the database manually 
+   for future reference if the data is important to you.
+ * [Forseti v2 configuration]({% link _docs/latest/configure/forseti/index.md %}) is different than v1 so 
+   you can not replace the v2 configuration file with the v1 configuration file.
+ * Configuration of the resources that are inventoried is not configureable.
 
-{% capture 1x_upgrade %}
 
-## Important caveats
+## Upgrading your v1 instance
+
+The suggested way of upgrading your Forseti instance is to do create a new project, [install 
+the latest v2 instance]({% link _docs/latest/setup/install.md %}), move all the rule files 
+from the v1 bucket to the v2 bucket.
```

Comment:
We should add at the end, that they should let v1 run side-by-side and then shutdown when they are comfortable that v2 is working properly.

Reply:
Updated